ol8/http/ol8.ks.in

```diff
@@ -0,0 +1,70 @@
+url --url="https://yum.oracle.com/repo/OracleLinux/OL8/baseos/latest/x86_64" ${KS_PROXY}
+poweroff
+firewall --enabled --service=ssh
+firstboot --disable
+ignoredisk --only-use=vda
+lang en_US.UTF-8
+keyboard us
+network --device eth0 --bootproto=dhcp
+firewall --enabled --service=ssh
```

Comment:
Same as line 3?

Reply:
@alexsander-souza any opinion/hint on this duplication for existing rhel/centos templates?

Reply:
I think it's a duplication, the scripts is like this since the dawn of time, so it doesn't look like a fix/workaround of a known bug. We can push another PR to remove this from existing templates